english/TheTaleOfTheProgrammer

```diff
@@ -0,0 +1 @@
+once upon a time there was a random 14 year old kid who decided to learn git and github. little did he know that the pulling request bullshit would fuck his brain.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.